### PR TITLE
paper: add joss draft and preprint

### PR DESCRIPTION
**Pull request by HLasse**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -139,3 +139,8 @@ dmypy.json
 .DStore
 
 .vscode
+paper/paper_quarto_files/*
```
Comment (KennethEnevoldsen) [inline .gitignore:142]:
```suggestion
# Paper rendering
paper/paper_quarto_files/*
```

File: paper/paper_quarto.html [hunk 2]
```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```
Comment (KennethEnevoldsen) [inline paper/paper_quarto.html:1]: A this an HTML version of it? (we could try to add it to the website)?

File: paper/paper_quarto.qmd [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
```
Comment (KennethEnevoldsen) [inline paper/paper_quarto.qmd:1]: is this autogenerated from the md or do we need to copy paste this every time?

**Review by KennethEnevoldsen — COMMENTED** [review]
If some of these files are autogenerated (e.g. .tex) it might be worth not including them?

File: .gitignore (hunk 1 above)
Comment (HLasse) [reply .gitignore:142]: the rendering of the .qmd files makes a HTML, pdf, and tex file. Yep, no reason to include the tex file.

File: paper/paper_quarto.qmd (hunk 3 above)
Comment (HLasse) [reply paper/paper_quarto.qmd:1]: the yaml preamble is different than the `paper.md` but the rest is exactly the same. So we can just copy-paste in the content of the matter and leave the preamble alone

File: paper/paper_quarto.html (hunk 2 above)
Comment (HLasse) [reply paper/paper_quarto.html:1]: yep, html version. Good idea - i'll try adding it to the docs!

File: paper/paper_quarto.qmd (hunk 3 above)
Comment (KennethEnevoldsen) [reply paper/paper_quarto.qmd:1]: Yea, saw it was stated in the readme so that is totally fine

**Comment by HLasse** [comment]
Paper html has been added to the docs. Will merge after upload to arxiv to get the correct link in the docs

**Pull request merged**